cmd/criticize/main.go

```diff
@@ -98,9 +101,19 @@ func parseArgv(l *linter) {
 		blame("-withOpinionated used with -enable=%q", *enable)
 	}
 
+	fmt.Printf("`%v` `%v`\n", *enable, *disable)
+
 	switch *enable {
 	case enableAll:
-		// Special case. l.enabledCheckers remains nil.
+		for _, rule := range lint.RuleList() {
+			if rule.Experimental && !l.withExperimental {
+				continue
+			}
+			if rule.VeryOpinionated && !l.withOpinionated {
+				continue
+			}
+			l.enabledCheckers = append(l.enabledCheckers, rule.Name())
+		}
 	case "":
```

Comment:
Do we need this after introducing `-disable` ?

Reply:
I guess not.